tests/test_simplify.py

```diff
@@ -159,6 +159,47 @@ def test_to_clifford_normal_form_graph(self):
             g0 = g.copy()
             to_clifford_normal_form_graph(g)
             self.assertTrue(compare_tensors(g0, g, preserve_scalar=True))
+    
+    def test_copy_simp(self):
+        g = Graph() 
+
+        v0 = g.add_vertex(VertexType.Z, 0, 0)
+        v1 = g.add_vertex(VertexType.X, 0, 1)
+        g.add_edge((v0, v1))
+        v2 = g.add_vertex(VertexType.BOUNDARY, 0, 3)
+        g.add_edge((v1, v2))
+        v3 = g.add_vertex(VertexType.BOUNDARY, 1, 3)
+        g.add_edge((v1, v3))
+        g1 = g.copy()
+        to_gh(g1)
+        copy_simp(g1)
```

Comment:
You should probably also assert that copy_simp actually simplified the graph, and that this is not a 'noop'.